eval/eval_test.go

```diff
@@ -211,16 +230,34 @@ func TestEval(t *testing.T) {
 			})
 			assert.NoError(t, err)
 
+			environmentName := e.Name()
+			overridesExists, err := doesFileExist(overridesPath)
+			assert.NoError(t, err)
+
+			if overridesExists {
+				var overrides testOverrides
+				expectedBytes, err := os.ReadFile(overridesPath)
+				require.NoError(t, err)
+				dec := json.NewDecoder(bytes.NewReader(expectedBytes))
+				dec.UseNumber()
+				err = dec.Decode(&overrides)
+				require.NoError(t, err)
+
+				if overrides.RootEnvironment != "" {
+					environmentName = overrides.RootEnvironment
+				}
+			}
```

Comment:
can probably simplify this a bit:

```suggestion
			environmentName := e.Name()
			overridesBytes, err := os.ReadFile(overridesPath)
			require.True(t, err == nil || errors.Is(err, os.ErrNotExist))
			if err == nil {
				var overrides testOverrides
				err = json.Unmarshal(overridesBytes, &overrides)
				require.NoError(t, err)
				environmentName = overrides.RootEnvironment
			}
```

then we can remove the `doesFileExist` function as well.

Reply:
makes sense